presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -108,6 +108,21 @@ The properties that apply to Hive connector security are listed in the
 :doc:`/connector/hive-security` section for a more detailed discussion of the
 security options in the Hive connector.
 
+HDFS Permissions
+^^^^^^^^^^^^^^^^
+Before running any ``CREATE TABLE`` or ``CREATE TABLE ... AS`` statements
```

Comment:
Call this `CREATE TABLE AS` since that's what we call it elsewhere in the documentation.